.travis.yml

```diff
@@ -46,12 +46,14 @@ matrix:
            CHECK_MBSTRING=0
   allow_failures:
     - php: "hhvm"
+    - php: "nightly"
     - php: "master"
+    - php: "7.4"
```

Comment:
The `php-7.4` need to require `vimeo/psalm` `3.x` version at least.

When using the `vimeo/psalm` `3.x` version, it will output the analyze error message.

And some analyze error message is not presented on `0.x`, `1.x` and `2.x` versions.

At this moment, the `php-7.4` version is allowed to be failure temporarily during Travis CI build.